hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/PeerCache.java

```diff
@@ -155,7 +155,7 @@ public Peer get(DatanodeID dnId, boolean isDomain) {
 
   private synchronized Peer getInternal(DatanodeID dnId, boolean isDomain) {
     List<Value> sockStreamList = multimap.get(new Key(dnId, isDomain));
-    if (sockStreamList == null) {
+    if (sockStreamList.isEmpty()) {
       return null;
```

Comment:
We can just drop this if check itself, the below logic can safely handle an empty list

Reply:
Thanks @ayushtkn for your comment.
Update PR, please help me review it again, thanks~